cmd/kogito/command/install/data_index.go

```diff
@@ -111,6 +111,11 @@ func (i *installDataIndexCommand) RegisterHook() {
 				log.Infof("infinispan-url informed. Infinispan will NOT be provisioned for you. Make sure that %s url is accessible from the cluster", i.flags.infinispan.ServiceURI)
 				i.flags.infinispan.UseKogitoInfra = false
 			}
+			if len(i.flags.kafka.ExternalURI) > 0 {
+				log.Infof("kafka-url informed. Kafka will NOT be provisioned for you. Make sure that %s url is accessible from the cluster", i.flags.kafka.ExternalURI)
+			} else if len(i.flags.kafka.Instance) > 0 {
+				log.Infof("kafka-instance informed. Kafka will NOT be provisioned for you. Make sure Kafka custom resource %s is properly deployed in the project", i.flags.kafka.Instance)
+			}
```

Comment:
In case no Kafka information delivered, I would add the information:

"No Kafka information has been given. It will be searched into the namespace"